.editorconfig

```diff
@@ -1,11 +1,204 @@
+# editorconfig.org
+
+# top-most EditorConfig file
 root = true
 
 [*]
-indent_size = 2
+trim_trailing_whitespace = true
+
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
+[**]
+insert_final_newline = true
 indent_style = space
+indent_size = 4
```

Comment:
this should be 2 by default, also sorry I didn't meant to promote this to the top level yet just that I thgink you could have removed `root=true` and the properties we already defined in there.